source/datasets/alpinebits.rst

```diff
@@ -0,0 +1,12 @@
+.. alpinebits
+
+This endpoint provides data from the `AlpineBits Open Standard
+<https://www.alpinebits.org>`_, whose purpose it to ease data exchange among
+actors involved in the alpine tourism.
+   
+==============  ========================================================
+Output          JSON, mime-type application/json
+E-mail contact  help@opendatahub.bz.it
+API version     v1
+Swagger URL     https://alpinebits.opendatahub.bz.it/AlpineBits
```

Comment:
Since there is no Swagger should add the link to the documentation here, not only in the Howto section:
https://www.alpinebits.org/developers/

Also the page to the development tools
http://development.alpinebits.org/#/home